### PR TITLE
[MIG ] 10.0 mig keychain

### PR DESCRIPTION
**Pull request by EBII**

### CODE REVIEW

File: keychain/tests/test_keychain.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ def setUp(self):
 
         self.keychain = self.env['keychain.account']
         config['keychain_key'] = Fernet.generate_key()
+        config['running_env'] = ""
```
Comment (hparfr) [inline keychain/tests/test_keychain.py]: It makes the line 26 useless.
Instead : 
`self.old_running_env = config.get('running_env', '')`

**Review by hparfr — CHANGES_REQUESTED** [review]
A small fix is needed for the tests.

**Comment by bealdav** [comment]
:+1:

**Review by hparfr — APPROVED** [review]

**Comment by hparfr** [comment]
👍

**Pull request merged**